app/views/organization_users/forms/_existing_user_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= simple_form_for organization_user, wrapper: :front_form do |f| %>
+  <%= card_body add_class: "border-top" do %>
+    <%= f.hidden_field :organization_id, value: organization.id %>
+    <div class="row">
+      <div class="col-md-6 mb-3 mb-md-0">
+        <%= f.input(:user_id, label: "Name", collection: users_for_select, include_blank: "None",
```

Comment:
This is probably a security hole for us. Organizations should not be able to see a list of users in our system.

We should probably just have them enter the email and if it matches an existing user, use that instead.